packages/office-ui-fabric-react/src/components/Rating/docs/new/RatingOverview.md

```diff
@@ -0,0 +1,5 @@
+Ratings show people’s opinions of a product, helping people make more informed purchasing decisions. People can also rate products they’ve purchased.
```

Comment:
Ratings can be used for things other than products and purchasing decisions. An example that comes to mind is rating the quality of content, but I know there are others.